tools/installers/containerd.sh

```diff
@@ -27,6 +27,12 @@ else
   declare -r CRITOOLS_VERSION=${CRITOOLS_VERSION:-1.18.0}
 fi
 
+# containerd < 1.4 doesn't work with cgroupv2 setup, so we check for that here
+if [[ $(mount | grep "/sys/fs/cgroup type cgroup2 ") && "${CONTAINERD_MAJOR}" -eq 1 ]] && [[ "${CONTAINERD_MINOR}" -le 4 ]]; then
```

Comment:
The comment says that it has to be less than 4, but -le means less than or equal. What is right here?